owners.md

```diff
@@ -14,4 +14,12 @@ If the Deploy workflow fails because you forgot to bump the `package.json` versi
 
 ### Information on publishing VSCode extensions
 
-Publishing is done by `vsce`. See https://code.visualstudio.com/api/working-with-extensions/publishing-extension. The Azure DevOps personal access token with permissions for publishing to the Visual Studio Marketplace **only** needs to be [stored as a repository secret](https://help.github.com/en/actions/configuring-and-managing-workflows/creating-and-storing-encrypted-secrets), and will need updating on expiry.
+Publishing is done by the tool `vsce` using a personal access token (PAT) for the Cylc "Azure DevOps organisation".
+
+#### Updating the PAT
+
+This needs to be done when the existing one expires. See https://code.visualstudio.com/api/working-with-extensions/publishing-extension for details on publishing, and how to create a PAT in Azure DevOps. Create a PAT with permissions for publishing to the Visual Studio Marketplace **only**, and [store it as a repository secret](https://help.github.com/en/actions/configuring-and-managing-workflows/creating-and-storing-encrypted-secrets).
```

Comment:
What causes a PAT to expire?
Are they time limited?

Reply:
Yes, up to 1 year